src/esp/bindings/ConfigBindings.cpp

```diff
@@ -24,8 +24,14 @@ py::object getObjectForConfigValue(const ConfigValue& value) {
       return py::cast(value.get<double>());
     case ConfigStoredType::String:
       return py::cast(value.get<std::string>());
+    case ConfigStoredType::MagnumColor3:
+      return py::cast(value.get<Mn::Color3>());
     case ConfigStoredType::MagnumVec3:
       return py::cast(value.get<Mn::Vector3>());
+    case ConfigStoredType::MagnumVec4:
+      return py::cast(value.get<Mn::Vector4>());
+    case ConfigStoredType::MagnumColor4:
+      return py::cast(value.get<Mn::Color4>());
```

Comment:
What I do in most APIs (SceneData, MeshData, MaterialData) is treating `Vector4` and `Color4` as the same type, and same for the three-component type. There's no reason not to, it's three/four numbers in both cases, and it reduces the API surface area quite significantly.

Reply:
That should be possible with the Vector type as well, the rgb(a) accessors are defined there already, next to xyz(w). What Color has on top is mainly colorspace conversion (sRGB, HSV, XYZ).

But nevertheless, what I meant was that you *can* still use `Color4` to both set and query a value, just that it'd be internally treated the same as `Vector4`. Which also implies that you can set/query `Vector4` values as `Color4` and vice versa, and it doesn't cause any error.

Reply:
Oh, yeah, I remember that.  Ok, so me saving this as a color3 and color4 is needlessly redundant. I'll remove those two. Thanks!

Reply:
...but keep the accessors, so folks can get Vector3 and Vector4 elements as Color3 and Color4 elements, if they wish.